ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/sno/cucushift-installer-rehearse-aws-ipi-disconnected-private-sno-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-sno
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-disconnected-private-sno-provision
+    post:
+    - chain: cucushift-installer-rehearse-aws-ipi-disconnected-private-sno-deprovision
+  documentation: |-
+    This is the workflow to trigger Prow's rehearsal test when submitting installer steps/chain/workflow
```

Comment:
this sentence seems pretty generic.  Can you adjust to document the purpose of the `cucushift-installer-rehearse-aws-ipi-disconnected-private-sno` workflow specifically, so folks reading the paragraph will understand how this workflow is different from other multi-step workflows?